test/langtools/tools/javac/annotations/typeAnnotations/classfile/Patterns.java

```diff
@@ -28,8 +28,8 @@
  * @modules java.compiler
  *          jdk.jdeps/com.sun.tools.javap
  * @build toolbox.JavapTask
- * @compile --enable-preview -source ${jdk.version} Patterns.java
- * @run main/othervm --enable-preview Patterns
+ * @enablePreview
+ * @run main/othervm Patterns
```

Comment:
do we still need `/othervm` ?

Reply:
I suppose it isn't strictly required; I assume the @enablePreview functionality handles this if needed. I'll go through and remove the /othervm options and make sure the tests still pass, etc.